views.py

```diff
@@ -107,3 +107,8 @@ def tofu(request):
     'View for tofu page.'
     return render(request, 'tofu.html', {'title' : 'Tofu',
                                          'tagline' : 'Not just for eating'})
+
+def about(request):
+    'View function for about page of site.'
+    return render(request, 'about.html', {'title' : 'About',
+                                          'tagline' : 'Get to know Us'})
```

Comment:
Let's make this title case.  Something I also changed in my change list yesterday.   😅  'to' can stay lowercase.

Reply:
It look like comment highlights a couple lines so it's not very clear. Opps.  I was talking about the tagline here. :)